agent/bench-scripts/pbench-fio.md

```diff
@@ -339,3 +339,27 @@ Here are the pbench-fio parameters:
 * **--numjobs=uint** - the total number of "jobs" to run.  This is typically set to the number of clients above.  Typically in the job file, you use numjobs=1.
 * **--pre-iteration-script=path** - this points to a local executable script (doesn't have to be bash) that will execute before each sample is run.  For an example of how this can be useful, see cache-dropping section above.
 * **--max-stddev=uint** - maximum percent deviation (100.0 \* standard deviation / mean) allowed for a successful test.  If you get test failures, consider either lengthening your test, changing test procedure to improve repeatability, or increasing this parameter.  Default is 5%.
+
+
+## firewalls
+
+*You must ensure that the network firewall is not up, or poke holes in the firewall
+for pbench-fio when using remote clients*.  Typically there are two possible
+firewall implementations encountered:
+
+* the **firewalld** service
+* the **iptables** service
+
+To temporarily disable (this may give security folks heartburn):
```

Comment:
Ha!

Reply:
This line was taken from `pbench-uperf.md`.